Tb birth place

fredkingham opened the pull request:
Moves Demographics.birth_place to Nationality.birth_place.

I haven't changed any of the advanced search as there is already Demographics.gp_practice code that is there and not populated.

Commit:
Creates Nationality.birth_place and copies the data from Demographics.birth_place

Commit:
Changes the front end to populate TB.birth_place to be

davidmiller commented on plugins/tb/migrations/0060_auto_copy_over_birth_place.py:
```diff
@@ -0,0 +1,37 @@
+
+    operations = [
+        migrations.RunPython(
+            migrate_forwards
```
Will this now stack trace if I try to go backwards?

fredkingham replied:
It did, I've actually changed it so that the reverse will copy data back from the tb nationality model as presumably that's what we want to theoretically undo this change (although we would probs do it in a seperate migration if so)

Commit:
Allow reverse code for copying over birth place

In theory if we are reversing this we are backing out the move to
keeping the data in nationality so copy it back over to demographics
